src/contract_wrappers/modules/sto/usd_tiered_sto_wrapper/common.ts

```diff
@@ -26,7 +20,7 @@ import {
   weiToValue,
 } from '../../../../utils/convert';
 import functionsUtils from '../../../../utils/functions_utils';
-import GeneralTransferManagerWrapper from '../../transfer_manager/general_transfer_manager_wrapper';
+import { GeneralTransferManager_3_0_0 } from '../../transfer_manager/general_transfer_manager_wrapper';
```

Comment:
This should be the common GTM wrapper

Reply:
I'm confused about this. I tried using common in place of 3.0.0 and it didn't work.